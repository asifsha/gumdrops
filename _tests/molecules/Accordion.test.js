/* globals mount */
import React from 'react';
import Accordion from '../../components/molecules/Accordion';

describe('Expect <Accordion>', () => {
    it('to render', () => {
        const props = {
            context: 'primary',
            size: 'sm',
            className: 'cool-stuff'
        };
        const wrapper = mount(
            <Accordion {...props}>
                <li>Accordion Item 1</li>
                <li>Accordion Item 2</li>
                <li>Accordion Item 3</li>
            </Accordion>
        );
        expect(wrapper).toMatchSnapshot();
    });
});
