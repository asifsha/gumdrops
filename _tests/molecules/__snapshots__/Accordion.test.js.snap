// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expect <Accordion> to render 1`] = `
<Accordion
  className="cool-stuff"
  context="primary"
  size="sm"
>
  <div
    className="gds-accordion cool-stuff gds-accordion--primary"
  >
    <ul
      className="gds-accordion-list"
    >
      <li
        context="primary"
        key=".0"
        size="sm"
      >
        Accordion Item 1
      </li>
      <li
        context="primary"
        key=".1"
        size="sm"
      >
        Accordion Item 2
      </li>
      <li
        context="primary"
        key=".2"
        size="sm"
      >
        Accordion Item 3
      </li>
    </ul>
  </div>
</Accordion>
`;
